pkgs/development/libraries/libAfterImage/default.nix

```diff
@@ -54,6 +54,13 @@ stdenv.mkDerivation {
       # workaround '-p0' patchflags below.
       stripLen = 1;
     })
+
+    # fix https://github.com/root-project/root/issues/10990
+    (fetchpatch {
+      url = "https://github.com/root-project/root/pull/11243/commits/e177a477b0be05ef139094be1e96a99ece06350a.diff";
```

Comment:
```suggestion
      url = "https://github.com/root-project/root/commit/e177a477b0be05ef139094be1e96a99ece06350a.patch";
```

Reply:
.patch includes extra bytes that we strip anyway
i don't know why it is needed to strip the pull/ from url, and you haven't explained why

Reply:
True, file ending does not matter. I moved the PR link into the comment because then you can just click on it and be where you want. If you click on the patch link then github redirects you to a link without the pull request.

Reply:
I usually just select the `https://github.com/root-project/root/pull/11243` to get to PR
or `https://github.com/root-project/root/commit/e177a477b0be05ef139094be1e96a99ece06350a` to get to the commit (e.g. to see github username).
I agree that having clickable links in the code where possible is a good idea. In this PR, there is a clickable link to the upstream issue and the rest can be reached by following that.